samples/python/qrcode.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import print_function
+import cv2 as cv
+import numpy as np
+import sys
+from glob import glob
+PY3 = sys.version_info[0] == 3
+import argparse
+
+if PY3:
+    xrange = range
+
+def str2bool(v):
+    if isinstance(v, bool):
+        return v
+    elif v.lower() in ('true'):
+        return True
+    elif v.lower() in ('false'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+#Input From the camera is to be implemented
+
+#Output Filename and It's extension
+o_f_name=''
+o_f_ext=''
+
+#Default values of input parameters
+parser = argparse.ArgumentParser(description='This program detects the QR-codes input images using OpenCV Library.')
+parser.add_argument('-i','--input',help="input image path (default input file path is 'opencv_extra/testdata/cv/qrcode/multiple/*_qrcodes.png",default="../../../opencv_extra/testdata/cv/qrcode/multiple/*_qrcodes.png",metavar="")
+parser.add_argument('-d','--detect',help="detect QR code only (skip decoding) (value True/False default value is False)",metavar='',type=str2bool,default=False)
+parser.add_argument('-m','--multi',help="use detect for multiple qr-codes (value True/False default value is True)",metavar="",type=str2bool,default=True)
+parser.add_argument('-o','--out',help="path to result file (default output filename is qr_code.png)",metavar="",default="qr_code.png")
+args = parser.parse_args()
+print(args)
```

Comment:
"global" code is not allowed in OpenCV Python samples (see "demo.py", it loads these files as modules).
Move such code into the `main()` function.

Reply:
> Move such code into the main() function

Not resolved.

Reply:
I have updated the changes please check